core/src/integration-test/java/com/scalar/db/util/JdbcAdminTestUtils.java

```diff
@@ -0,0 +1,60 @@
+package com.scalar.db.util;
+
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.storage.jdbc.JdbcAdmin;
+import com.scalar.db.storage.jdbc.JdbcConfig;
+import com.scalar.db.storage.jdbc.JdbcUtils;
+import com.scalar.db.storage.jdbc.RdbEngine;
+import com.scalar.db.storage.jdbc.query.QueryUtils;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import org.apache.commons.dbcp2.BasicDataSource;
+
+public class JdbcAdminTestUtils extends AdminTestUtils {
+
+  private final RdbEngine rdbEngine;
+  private final JdbcConfig config;
+
+  public JdbcAdminTestUtils(Properties properties) {
+    super();
+    config = new JdbcConfig(new DatabaseConfig(properties));
+    metadataNamespace = config.getTableMetadataSchema().orElse(JdbcAdmin.METADATA_SCHEMA);
+    metadataTable = JdbcAdmin.METADATA_TABLE;
+    rdbEngine = JdbcUtils.getRdbEngine(config.getJdbcUrl());
+  }
+
+  @Override
+  public void dropMetadataTable() throws SQLException {
+    execute(
+        "DROP TABLE "
+            + QueryUtils.enclosedFullTableName(metadataNamespace, metadataTable, rdbEngine));
+
+    String dropNamespaceStatement;
+    if (rdbEngine == RdbEngine.ORACLE) {
+      dropNamespaceStatement = "DROP USER " + QueryUtils.enclose(metadataNamespace, rdbEngine);
+    } else {
+      dropNamespaceStatement = "DROP SCHEMA " + QueryUtils.enclose(metadataNamespace, rdbEngine);
+    }
+    execute(dropNamespaceStatement);
+  }
+
+  @Override
+  public void truncateMetadataTable() throws Exception {
+    String truncateTableStatement =
+        "TRUNCATE TABLE "
+            + QueryUtils.enclosedFullTableName(metadataNamespace, metadataTable, rdbEngine);
+    execute(truncateTableStatement);
+  }
+
+  @SuppressFBWarnings("OBL_UNSATISFIED_OBLIGATION")
```

Comment:
I don't think we will need this after merging the latest changes from master into this branch.

Reply:
Ok, I updated the `spotbugs-exclude.xml` file to add a matching file regex for `XXXTestUtils.java` files.

Reply:
Oh sorry, I thought we didn't need to update `spotbugs-exclude.xml` to suppress the warning.